lib/Runtime/Math/AsmJsMath.inl

```diff
@@ -71,6 +71,20 @@ namespace Js
         return maxCheckNan(aLeft, aRight);
     }
 
+    template<>
+    inline double AsmJsMath::Abs<double>(double aLeft)
```

Comment:
That's right.

---
In reply to: [89415359](https://github.com/Microsoft/ChakraCore/pull/2050#discussion_r89415359) [](ancestors = 89415359)